packages/platform-android/src/commands/runAndroid/runOnAllDevices.js

```diff
@@ -39,6 +39,10 @@ function runOnAllDevices(
     const tasks = args.tasks || ['install' + toPascalCase(args.variant)];
     const gradleArgs = getTaskNames(args.appFolder, tasks);
 
+    if (args.port != null) {
```

Comment:
We should probably use stricter equal operator `!==`. Or is it used on purpose here?

Reply:
I figured the value could be undefined if the argument wasn't provided, but I also wasn't sure if it could ever be null too.

I bet I could make this `"port" in args` or `args.port !== undefined`

Reply:
`args.port` can be: `undefined`, `NaN` (we're passing string input through `Number()`), `number`

Reply:
I guess `if(args.port)` would be less confusing in that case :D